source/features/rgh-linkify-yolo-issues.tsx

```diff
@@ -0,0 +1,27 @@
+import * as pageDetect from 'github-url-detection';
+
+import React from 'dom-chef';
+
+import select from 'select-dom';
+
+import features from '../feature-manager';
+
+import {wrap} from '../helpers/dom-utils';
+
+import {getRghIssueUrl} from '../helpers/rgh-issue-link';
+
+function init(): void {
+	for (const issueCell of select.all('td:nth-child(3)')) {
```

Comment:
Can you find a more specific selector? This will find all the tables on the page (if any)

Reply:
I have added this `.js-csv-data` selector. It works but not sure if this is correct selector.